### ISSUE TITLE
Problems when using the splitter control

### ISSUE THREAD

**Issue by W98X** [XusinboyBekchanov/VisualFBEditor#93]
What I really like about VisualFBEditor is that it has a splitter control. Which is something important that Firefly and WinFBE don't have.

However, I have noticed there are some problems when using the splitter control. Which I will now list below:

### Problem 1. 
When the splitter is being dragged, there are lots of visual artifacts. As shown in the pictures below:

Horizontal splitter being dragged:
![vfbe-horizontal-splitter-glitches](https://user-images.githubusercontent.com/88371717/128339528-470b4a76-0845-4490-9f2f-e4a5bbd6baaa.png)

Vertical splitter glitches whilst dragging:
![vfbe-splitter-vertical-artifacts](https://user-images.githubusercontent.com/88371717/128339699-568f029f-20ca-4925-ae93-abb6f6074527.png)

### Problem 2. 
The splitter can be dragged off past the edge of the form so that it is no longer able to be seen or operated. The only way to recover from this is to resize the form to make it bigger and goes past where the splitter is.
![vfbe-splitter-vertial-extreme-edge](https://user-images.githubusercontent.com/88371717/128339810-ec3cfc4b-d91a-4eb5-aed8-62622e25ad3d.png)

### Problem 3. 
The property editor for the splitter says the alignment of the splitter as a number and not as a word. ie "horizontal" or "vertical" or "left", "right", "top", "bottom".
![vfbe-splitter-alignment-property](https://user-images.githubusercontent.com/88371717/128339889-2afc3914-2376-4844-99db-2efd32a5de97.png)

**Comment by XusinboyBekchanov** [comment]
Fixed: Update controls on drag Splitter
Fixed: Splitter not drags bigger than parent
Added: SplitterAlignmentConstants enum
https://github.com/XusinboyBekchanov/MyFbFramework/commit/5afe5edfb8bd740fcbe6d1191debd484805a90f5

**Comment by W98X** [comment]
Yes, the splitter doesn't make artifacts when dragging the splitter now! Big improvement, looks much better now :grinning:

And the splitter doesn't go past the edge of the form anymore. :thumbsup:

However, I have noticed that other software's splitters doesn't go so close to the edge, but stop about 20 pixels before the edge to make it easier to click on the splitter. For example CHM helps files vertical splitter doesn't go all the way to the edge and neither does anything made in .net or newer versions of Delphi.

Can VisualFBEditor's splitter be made to stop before reaching the edge?

Here's a screenshot of a test I did in vb.net and it won't let me drag the splitter anymore further to the right than this:
![vbnet-splitter-edge-limits](https://user-images.githubusercontent.com/88371717/128478532-6193c308-91a8-497b-9507-a13f359eab0a.png)
